config.yaml

```diff
@@ -57,6 +57,8 @@ languages:
         url: /learn
       - title: Array computing
         url: /arraycomputing
+      - title: Case Studies
+        url: /case-studies/blackhole-image
```

Comment:
Can you remove this again (also the two entries below)? They're not needed (and the link is anyway only to one of the two case studies), and it's easier to remove now rather than later.

Reply:
ok done